eth/backend.go

```diff
@@ -555,7 +555,7 @@ func (s *Ethereum) Miner() *miner.Miner { return s.miner }
 func (s *Ethereum) Name() string                         { return s.net.Name }
 func (s *Ethereum) AccountManager() *accounts.Manager    { return s.accountManager }
 func (s *Ethereum) BlockChain() *core.BlockChain         { return s.blockchain }
-func (s *Ethereum) BlockProcessor() *core.BlockProcessor { return s.blockProcessor }
+func (s *Ethereum) BlockValidator() *core.BlockValidator { return s.blockValidator }
```

Comment:
Because of a `debug` command. Though that reminds me I still need to patch that up after the changes.